.github/workflows/sync-version-with-upstream.yml

```diff
@@ -0,0 +1,30 @@
+name: Update
+
+on:
+  # Runs at 10:00 UTC every day
+  schedule:
+    - cron:  '0 10 * * *'
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  sync:
+    name: 🔄 Sync version with upstream
+    environment: "Edge Branch"
+    runs-on: ubuntu-latest
+    steps:
+      - name: 🔄 Sync version with upstream
+        uses: snapcrafters/ci/sync-version@main
```

Comment:
Could we pin the version based on hash like the other actions we are using?

Reply:
I think so, but I would expect dependabot to take care of that.

Reply:
I think StepSecurity can do this for us here: https://app.stepsecurity.io/secureworkflow, but I don't think dependabot will do it automotically.